src/app/components/Book/Book.jsx

```diff
@@ -114,11 +114,12 @@ const Book = ({ pick, isJsEnabled }) => {
 
   const renderTags = (tags, jsEnabled) => {
     const tagsMarkup = !_isEmpty(tags) ?
-      tags.map((tag, i) => <span key={i}>{tag}{i !== (tags.length - 1) ? ', ' : ''}</span>) : null;
+      tags.map((tag, i) => <li key={i}><span>{tag}{i !== (tags.length - 1) ? ', ' : ''}</span></li>) : null;
```

Comment:
You don't need the `span`. You can add the CSS for the span directly to the li.

Reply:
What exactly is the function of the `span` tags here and around the "Tags" heading?